tests/helpers/acceptance.js

```diff
@@ -55,6 +57,15 @@ function createTestTargets(projectName, options) {
   options.command = options.command || 'new';
 
   return applyCommand(options.command, projectName, '--skip-npm', '--skip-bower', `--directory=${outputDir}`)
```

Comment:
This should be fixed by https://github.com/ember-cli/ember-cli/pull/8432. Thanks for investigating and reporting the issue.